### DISCUSSION TITLE
How to control browser FPS?

### DISCUSSION THREAD

**Discussion by zxcvqwerasdf** [cefsharp/CefSharp#3746] [Q&A] [answered]
Hello, i want to control browser frames per second (offscreen).
For now i using ScreenshotAsync in loop, there is any more perfomance way to do it? Thank you.

**Comment by amaitland** [comment, marked as answer]
See https://github.com/cefsharp/CefSharp/blob/cefsharp/92/CefSharp.OffScreen.Example/Program.cs#L56
